### PR TITLE
Interactive

### PR DESCRIPTION
**Pull request by ilyassatlassi**
In this project, I add some functionality to your application to make it interactive.

### CODE REVIEW

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @ilyassatlassi , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has not complete readme . Please make sure that your  readme has the future features and 1 - 3 potentials features . :+1: 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/modules/main.js [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+let index = 1;
+let todoList = [];
+
+const showList = () => {
+  let litItems = '';
+  todoList.forEach((element) => {
+    if (element.completed === true) {
+      litItems += `<li class="task">
+                                  <div>
+                                  <input type="checkbox" class="checkFunc" checked onclick="checkFunc(${element.index - 1});" id="check${element.index - 1}">
```
Comment (fobadara) [inline src/modules/main.js:14]: [REQUIRED] Kindly make this a change event listener instead of click

**Review by fobadara — CHANGES_REQUESTED** [review]
Hi @ilyassatlassi, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
✅ Linters are passing
✅ Good first attempt

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/main.js (hunk 1 above)
Comment (ilyassatlassi) [reply src/modules/main.js:14]: @fobadara please can you clarify i didn't understand what you mean

**Review by rotimiazeez — APPROVED** [review]
Hi @ilyassatlassi 👏🏽, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlight
- No linter error 👍🏽
- Good commit messages 👍🏽 

## Optional suggestions
- [OPTIONAL] The logo provided in your README file is broken kindly fix it

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me(@rotimiazeez) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**